src/services/locale.ts

```diff
@@ -119,6 +119,27 @@ const locale = {
                 output: (value: string): string => value,
             },
         },
+        tenure: {
+            tenureType: {
+                field: 'Tenure Type',
+                output: (value: string): string => value ?? '[No entry]',
+            },
+            isActive: {
+                field: 'Tenure status',
+                output: (value: boolean): string =>
+                    value ? 'Active' : 'Inactive',
+            },
+            startOfTenureDate: {
+                field: 'Start date',
+                output: (value: string): string =>
+                    format(parseISO(value), 'dd/MM/yy'),
```

Comment:
This seems to be the only place where we don't show the full year. Can you check with Rob if thats desired?

Reply:
Will check with Rob